### ISSUE TITLE
Ignore property in Model builder entity for Update procedure only

### ISSUE THREAD

**Issue by jacontre** [dotnet/efcore#33676]
Is there a way to use InsertUsingStoredProcedure and UpdateUsingStoredProcedure with the same entity and configure it so that I can ignore a property on Update that is not used. Our example is using a CreatedDateTime and LastModifiedTime where both are used in the Insert procedure but then CreatedDateTime is not passed into the update procedure and we would like to ignore it for that case.

**Comment by AndriySvyryd** [comment]
This depends on https://github.com/dotnet/efcore/issues/14121

**Comment by bytefish** [comment]
@AndriySvyryd Is there any way to work around this issue, so we could ignore a field, when using `InsertUsingStoredProcedure`?

**Comment by AndriySvyryd** [comment]
@bytefish No, I can't think of a way around this currently, apart from adding a dummy parameter to the sproc.

**Comment by bytefish** [comment]
@AndriySvyryd Thanks for the confirmation and thanks for your work on EF Core! 👍🏻
